docker/base/install_pythons.sh

```diff
@@ -17,7 +17,7 @@ PYENV_VERSIONS=(
   3.9.19
   3.10.14
   3.12.3
-  3.13.0a6
+  3.13.0b1
```

Comment:
An item of note: this is the 1st Python release to support ZIP64 in zipimporter: aka Python 3.13 PEXes can now contain huge ML apps without `--layout {loose,packed}` shenanigans being necessary.